src/db.php

```diff
@@ -2425,39 +2425,42 @@ private function get_sqlite_version()
          * Method to call PDO::beginTransaction().
          *
          * @see PDO::beginTransaction()
-         * @return boolean
          */
-        public function beginTransaction()
+        public function beginTransaction(): bool
```

Comment:
Unfortunately, we can't use return types due to compatibility with PHP 5.6.

Reply:
Absolutely right. The solution will be to use the ReturnTypeWillChange annotation on all those methods. Jut sent a new update. Thanks for your reply!